app/views/workflow-ui/job/index.mustache.html

```diff
@@ -6,6 +6,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../../../css/eui.min.css">
     <link rel="stylesheet" href="../../../css/default.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link href="https://unpkg.com/@yaireo/tagify/dist/tagify.css" rel="stylesheet" type="text/css" />
```

Comment:
James and I were noticing that sometimes we wait some time for the UI to load and the bottom left of the browser says 'waiting for unpkg.com' - just want to sanity check this is the best way to load the CSS.

Reply:
Can we serve the CSS up via CloudFront?

Reply:
I'm not sure about CloudFront but am open to exploring that if necessary.